util/build.json

```diff
@@ -13,7 +13,8 @@
                 "asprintf.o",
                 "private.o",
                 "err.o",
-                "isatty.o"
+                "isatty.o",
+                "progname.o"
```

Comment:
since you deleted private.c, private.o needs to be removed. Please replace private.o with progname.o. This will keep the diff smaller. (you'll also need to re-run autogen.sh)